### PR TITLE
DAT-74149 attribution models and lookback window

### PR DESCRIPTION
**Pull request by itaydafna**
## Description
Range component enhancements and bug-fix: 

 - Add label and value renderers
- Add innerMax prop
-  Fix buggy onDragEnd when releasing inside the range-component.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Are you adding a new package?
- [ ] yes
- [x] no

### CODE REVIEW

File: src/components/Range.js [hunk 1]
```diff
@@ -16,12 +18,20 @@ export const Range = (props) => {
     showValue,
     className,
     onChange,
+    innerMax,
```
Comment (itaydafna) [inline src/components/Range.js:21]: dynamic inner boundary which doesn't allow dragging above this value

File: src/components/Range.js [hunk 2]
```diff
@@ -63,16 +80,29 @@ export const Range = (props) => {
...
-    [outerEl, value]
+    [outerEl, value, innerMaxPercentage]
   );
 
   return (
-    <Container disabled={disabled} className={className} onClick={handleClick}>
+    <Container
+      disabled={disabled}
+      className={className}
+      onMouseDown={handleClick}
```
Comment (itaydafna) [inline src/components/Range.js:104]: `onClick` was triggered by the `Draggable`'s `onMouseUp` ([preventDefault or stopPropagation do not solve this](https://stackoverflow.com/questions/8643739/cancel-click-event-in-the-mouseup-event-handler/8927598)).

This resulted in issues selecting a single value when releasing the draggable/thumb above the component (move apparent in large designs)

handling the click event `onMouseDown` solved this.

**Review by crazypixel — APPROVED** [review]

**Review by crazypixel — APPROVED** [review]

**Review by Chikel — APPROVED** [review]

**Pull request merged**